.editorconfig

```diff
@@ -10,6 +10,10 @@ insert_final_newline = true
 [*.{c,h,in,lua}]
 max_line_length = 100
 
+[*.py]
+indent_style = space
```

Comment:
Isn't this already specfied above?

Reply:
Yea I guess it’s already included in the wildcard section. Doesn’t hurt to be explicit imo but I don’t care either way.

@dundargoc it’s up to you

Reply:
shipit